common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -954,9 +954,11 @@ def test_user_can_navigate_to_members_profile_page(self):
         self._set_team_configuration_and_membership()
         self.team_page.visit()
 
+        learner_name = self.team_page.first_member_username
+
         self.team_page.click_first_profile_image()
 
-        learner_profile_page = LearnerProfilePage(self.browser, self.team_page.first_member_username)
+        learner_profile_page = LearnerProfilePage(self.browser, learner_name)
```

Comment:
@jzoldak I'm following up on the flaky tests investigation here: https://github.com/edx/edx-platform/pull/9361